CHANGES.md

```diff
@@ -0,0 +1,21 @@
+## v0.1.1
+
+Conduit 0.1.1 is focused on making it easier to get started with Conduit.
+
+* Conduit can now be installed on Kubernetes clusters that use RBAC
+* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2 protocols.
+* The CLI `tap` command output has been reformatted to be line-oriented, making it easier to parse with common UNIX command line utilities
```

Comment:
tioli

> The output of `conduit tap` has been reformatted to be more readable by machines and humans alike.

Reply:
This bullet point was already heavily reviewed, so I'm going to leave it.